recipes/nsync/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class NsyncConan(ConanFile):
+    name = "nsync"
+    homepage = "https://github.com/google/nsync"
+    description = "Library that exports various synchronization primitive"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("c", "thread", "multithreading", "google")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["CMakeLists.txt"]
+
+    options = {"with_tests": [True, False]}
+    default_options = {"with_tests": False}
+
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```

Is cmake_find_package really needed?

Reply:
to be honest IDK